### PR TITLE
ISPN-8232 Move more [Optimistic|Pessimistic]TxPatitionAndMergeDuring*…

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.jboss.org/browse/ISPN-8232

Obviously, leave the Jira open

### CODE REVIEW

**Comment by pruivo** [comment]
you miss one:
```
org.infinispan.partitionhandling.PessimisticTxPartitionAndMergeDuringPrepareTest.testPrimaryOwnerIsolatedPartition[DIST_SYNC, DENY_READ_WRITES]
```

File: core/src/test/java/org/infinispan/partitionhandling/OptimisticTxPartitionAndMergeDuringCommitTest.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
  * @author Pedro Ruivo
  * @since 8.0
  */
-@Test(groups = "functional", testName = "partitionhandling.OptimisticTxPartitionAndMergeDuringCommitTest")
+@Test(groups = "unstable", testName = "partitionhandling.OptimisticTxPartitionAndMergeDuringCommitTest", description = "ISPN-8232")
```
Comment (danberindei) [inline core/src/test/java/org/infinispan/partitionhandling/OptimisticTxPartitionAndMergeDuringCommitTest.java]: Should keep the original group as well

```suggestion
@Test(groups = {"functional", "unstable"}, testName = "partitionhandling.OptimisticTxPartitionAndMergeDuringCommitTest", description = "ISPN-8232")
```

Comment (tristantarrant) [reply core/src/test/java/org/infinispan/partitionhandling/OptimisticTxPartitionAndMergeDuringCommitTest.java]: Does that still prevent the test from being run by default ?

Comment (danberindei) [reply core/src/test/java/org/infinispan/partitionhandling/OptimisticTxPartitionAndMergeDuringCommitTest.java]: Yes

Comment (tristantarrant) [reply core/src/test/java/org/infinispan/partitionhandling/OptimisticTxPartitionAndMergeDuringCommitTest.java]: Done

**Comment by danberindei** [comment]
Integrated, thanks Tristan!

**Comment by danberindei** [comment]
Oops, I really thought I had integrated this :)

**Pull request merged**